src/test/java/seedu/mindmymoney/command/ListCommandTest.java

```diff
@@ -0,0 +1,16 @@
+package seedu.mindmymoney.command;
+
+import org.junit.jupiter.api.Test;
+import seedu.mindmymoney.MindMyMoneyException;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class ListCommandTest {
+
```

Comment:
Lecture says need to add in a happy path before a negative path, so perhaps can print a list that has input inside?